src/foam/dashboard/view/Card.js

```diff
@@ -97,7 +98,8 @@ foam.CLASS({
           }),
           height: this.slot(function(data$mode, height) {
             return data$mode == 'config' ? 'inherit' : ( height + 'px' );
-          })
+          }),
+          'background': ( this.data.obj && this.data.obj.color ) ? this.data.obj.color : /*%WHITE%*/ '#ffffff'
```

Comment:
The border is coming from this view. If I change the color on the child element, it will overlap the border
<img width="1440" alt="Screen Shot 2022-05-16 at 10 51 31 AM" src="https://user-images.githubusercontent.com/58699954/168621175-e9dbd6bf-ba48-426e-af2d-d1657a8d0766.png">
<img width="1433" alt="Screen Shot 2022-05-16 at 10 51 25 AM" src="https://user-images.githubusercontent.com/58699954/168621237-132f48b8-0e5a-41ee-9758-720f7054e7ce.png">

Reply:
Would rather duplicate the border-radius prop on the CitationView than add a specific prop to a generic view. So just adding `border-radius: 22px` to CardDashboardCitationView